frappe/__init__.py

```diff
@@ -1865,8 +1865,8 @@ def format(*args, **kwargs):
 
 
 def get_print(
-	doctype=None,
-	name=None,
+	doctype,
+	name,
```

Comment:
Is it necessary to make these mandatory? I guess this might break in various places (printing reports, printing arbitrary html, ...)

Reply:
You are correct. Since doc could be passed directly this would have broken the function signature. Fixed this. However  I will be making a new pull request with only these changes, as my fork is diverging on other aspects as well. (for which i will also make an issue).